config/environments/production.rb

```diff
@@ -54,15 +54,7 @@
   config.log_tags = [ :request_id ]
 
   # Use a different cache store in production.
-  config.cache_store = :dalli_store,
```

Comment:
Might want to double check we're not running any memcache stores anymore after merging this.  No sense wasting money.